content/es/docs/reference/glossary/namespace.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Namespace
+id: namespace
+date: 2018-04-12
+full_link: /docs/concepts/overview/working-with-objects/namespaces
+short_description: >
+  Abstracción utilizada por Kuberentes para soportar multples clústeres virtuales en el mismo clúster físico.
+aka:
+tags:
+- fundamental
+---
+
+Abstracción utilizada por Kuberentes para soportar multples clústeres virtuales en el mismo {{< glossary_tooltip text="clúster" term_id="cluster" >}} físico.
```

Comment:
```suggestion
Abstracción utilizada por Kubernetes para soportar multiples clústeres virtuales en el mismo {{< glossary_tooltip text="clúster" term_id="cluster" >}} físico.
```

“físico” or “real”?

Reply:
`Físico` is more common, and múltiples misses the acute accent. Fixed in https://github.com/kubernetes/website/pull/22259/commits/2e79d3e27ae4c9e7ccc4202fc7bcc5266b8c592c